e2e/tools/validator/src/validator/cases/__init__.py

```diff
@@ -6,25 +6,24 @@
 
 RAW_PROM_QUERIES = [
     {
-        "expected_query": "rate(kepler_process_package_joules_total{{pid='{vm_pid}', mode='dynamic'}}[{interval}])",
+        "expected_query": "rate(kepler_{level}_package_joules_total{{{query}, mode='dynamic'}}[{interval}])",
```

Comment:
Shouldn't this be {{level}}? Reason being that the { and } characters need to be escaped?